libs/sdk-backend-spi/src/organization/settings/index.ts

```diff
@@ -26,6 +26,24 @@ export interface IOrganizationSettingsService {
      */
     setLocale(locale: string): Promise<void>;
 
+    /**
+     * Sets timezone for organization.
+     *
+     * @param locale - for example "en-US", "cs-CZ", etc.
+     *
+     * @returns promise
+     */
+    setTimeZone(timezoneId: string): Promise<void>;
+
+    /**
+     * Sets date format for organization.
+     *
+     * @param formatId - example "en-US", "cs-CZ", etc.
+     *
+     * @returns promise
+     */
+    setDateFormat(formatId: string): Promise<void>;
```

Comment:
Should we change the param name `formatId` → `dateFormat`?

As the function name is `setDateFormat`, it would make more sense if the param is named `dateFormat`.

Reply:
fixed